src/main.rs

```diff
@@ -20,6 +20,9 @@ struct Args {
     /// path to the output directory
     #[arg(short, long)]
     output_directory: PathBuf,
+
+    #[arg(short, long, default_value = "false", default_missing_value = "true")]
```

Comment:
whats the difference between default_value and default_missing_value?

Reply:
`default_value` is the default value of the `force_overwrite` attribute of the struct when nothing is provided by the user. `default_missing_value` is the default value when the user passes the `-f` or `--force-overwrite` flag to the program but nothing else afterwards, so no `-f true` or `-f false` (which the user could do and is maybe confusing behavior).